receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -312,18 +313,77 @@ func getAge(start time.Time, end time.Time) int64 {
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
 	client graphql.Client,
+	rClient *github.Client,
 	repo string,
 ) (map[metadata.AttributeCveSeverity]int64, error) {
-	c, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
-	if err != nil {
-		return nil, err
+	d := ghs.getDbotAlerts(ctx, client, repo)
+	c := ghs.getCodeScanAlerts(ctx, rClient, repo)
+
+	return mapSeverities(d, c), nil
+}
+
+func (ghs *githubScraper) getDbotAlerts(
+	ctx context.Context,
+	client graphql.Client,
+	repo string,
+) []CVENode {
+
+	var all []CVENode
+	var cursor *string
+
+	for hasNextPage := true; hasNextPage; {
+		alerts, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo, cursor)
+
+		if err != nil {
+			ghs.logger.Sugar().Errorf("error %v getting dependabot alerts from repo %s", zap.Error(err), repo)
+			return nil
+		}
+
+		hasNextPage = alerts.Repository.VulnerabilityAlerts.PageInfo.HasNextPage
+		cursor = &alerts.Repository.VulnerabilityAlerts.PageInfo.EndCursor
+		all = append(all, alerts.Repository.VulnerabilityAlerts.Nodes...)
+
+	}
+
+	return all
+}
+
+// Get the Code Scanning Alerts count for a repository via the REST API
+func (ghs *githubScraper) getCodeScanAlerts(
+	ctx context.Context,
+	client *github.Client,
+	repoName string,
+) []*github.Alert {
+	var all []*github.Alert
+
+	// Options for Pagination support, default from GitHub was 30
+	// https://docs.github.com/en/rest/repos/repos#list-repository-contributors
```

Comment:
This comment points to the list contributors query. What's the default for the list alerts call? Should we set it to 100?

Reply:
Updated with a comment and link. Github defaults it to 30 calls but allows for a max of 100. We can definitely fine tune this as needed

Reply:
As discussed, lowered to 50. Marking as resolved